### DISCUSSION TITLE
How to disable chakra ui stories from appearing

### DISCUSSION THREAD

**Discussion by nickngqs** [storybookjs/storybook#18821] [Help] [answered]
When i install chakra ui, it automatically generates some stuff for chakra ui at the side. How can i disable this?

**Comment by jonniebigodes** [comment, marked as answer]
@nickngqs, if not mistaken, you can disable ChakraUI's stories by updating your Storybook configuration file (i.e., `.storybook/main.js`) as follows:
```js
// .storybook/main.js
module.exports = {
  stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
  staticDirs: ["../public"],
  addons: [
    "@storybook/addon-links",
    "@storybook/addon-essentials",
    "@storybook/addon-interactions",
    "@storybook/preset-create-react-app",
  ],
  framework: "@storybook/react",
  core: {
    builder: "@storybook/builder-webpack5",
  },
  refs: {
    "@chakra-ui/react": {
      disable: true, // 👈 chakra stories disabled here
    },
  },
};
````


By default, they're added through [package composition](https://storybook.js.org/docs/react/sharing/package-composition#configuring) to give you some information on how to use their components. Don't forget to restart your Storybook once you make this change.


Hope the answer helped you out.

Have a great weekend.

Stay safe

**Reply by nickngqs** [reply]
hey thanks alot! it works!
